packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -21,22 +21,38 @@ export const evaluationResults = latitudeSchema.table(
     evaluationId: bigint('evaluation_id', { mode: 'number' })
       .notNull()
       .references(() => evaluations.id, { onDelete: 'cascade' }),
+    // TODO: remove
     documentLogId: bigint('document_log_id', { mode: 'number' })
       .notNull()
       .references(() => documentLogs.id),
+    // TODO: remove
     providerLogId: bigint('provider_log_id', { mode: 'number' }).references(
       () => providerLogs.id,
     ),
+    evaluatedProviderLogId: bigint('evaluated_provider_log_id', {
+      mode: 'number',
+    }).references(() => providerLogs.id),
+    evaluationProviderLogId: bigint('evaluation_provider_log_id', {
+      mode: 'number',
+    }).references(() => providerLogs.id),
```

Comment:
Is this no the same as `providerLogId`? We could just replace the attribute name in drizzle, no need for a migration

Reply:
not backwards compatible